framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -618,10 +618,12 @@ protected void checkPostconditions(MethodTree node, ExecutableElement methodElem
                     // check anything
                 } else {
                     CFAbstractValue<?> value = exitStore.getValue(expr);
-                    AnnotationMirror inferredAnno =
-                            value == null
-                                    ? null
-                                    : value.getType().getAnnotationInHierarchy(annotation);
+                    AnnotationMirror inferredAnno = null;
```

Comment:
All three uses are in very similar methods for checking preconditions, postconditions, or conditional postconditions.  The right way to fix this would be to refactor all three methods, but this is unrelated to the changes in this pull request.